src/main/java/org/polyjdbc/core/query/InsertQuery.java

```diff
@@ -100,24 +86,11 @@ String getSequenceField() {
     public InsertQuery value(String fieldName, Object value) {
         valueNames.append(fieldName).append(", ");
         values.append(":").append(fieldName).append(", ");
-        query.setArgument(fieldName, value);
+        setArgument(fieldName, value);
         return this;
     }
 
-    /**
-     * Manually set sequenced field value.
-     */
-    public InsertQuery sequenceValue(long value) {
-        query.setArgument(sequenceField, value);
-        sequenceValueSet = true;
-        return this;
-    }
-
-    /**
-     * Returns true if sequenced field value was set manually and there is no
-     * need to use sequence generator.
-     */
-    public boolean isSequenceValueSet() {
-        return sequenceValueSet;
+    void setArgument(String fieldName, Object value) {
```

Comment:
Chyba nie rozumiem, nie dodawałem żadnych delegatorów. Co dokładnie 'breaks everything'?

Jedyna zmiana w API to usunięcie możliwości ustawiania ręcznie ID, więc 2 metod: `sequenceValue(long value)` i `isSequenceValueSet()`, ponieważ nie byłem w stanie ich zachować po wydzieleniu klasy `InsertWithAutoincrement`.

Te metody są a) niepotrzebne, b) nie mają sensu dla MySql, btw kto chciałby robić coś takiego ?

```
InsertQuery query = javersPolyJDBC.query().insert().into("my_table")              
                .value("my_field", 1)
                .sequence("my_id", "ANYTHING !")
                .sequenceValue(5);
```

Jeśli nie chcę używać sekwencji to po prostu ich nie używam i ustawiam ID wprost:

```
InsertQuery query = javersPolyJDBC.query().insert().into("my_table")              
                .value("my_field", 1)
                .value("my_id", "5");
```